rootfs/templates/kops/default.yaml

```diff
@@ -34,7 +34,8 @@ spec:
       idleTimeoutSeconds: {{ getenv "KOPS_API_LOAD_BALANCER_IDLE_TIMEOUT_SECONDS" "600" }}
   kubeAPIServer:
   {{- if bool (getenv "KOPS_AUTHORIZATION_RBAC_ENABLED" "false") }}
-    authorizationMode: {{ getenv "KOPS_KUBE_API_SERVER_AUTHORIZATION_MODE" "RBAC,AlwaysAllow" }}
+    anonymousAuth: false
```

Comment:
I think this should be moved above the conditional here as `anonymousAuth` should always be denied, no?

Reply:
In general, with RBAC off, our configuration is `AlwaysAllow`, so I do not think `anonymousAuth` has any effect. I prefer to keep it in the conditional to highlight that.